ca_gazebo/launch/create_apriltag.launch

```diff
@@ -0,0 +1,44 @@
+<launch>
+	<arg name="env" default="apriltags"/>
+  	<arg name="paused" value="$(optenv PAUSED false)"/>
+	<arg name="gui" value="$(optenv GUI true)"/>
+	<arg name="debug" value="$(optenv DEBUG false)"/>
+	<arg name="aprildebug" value="$(optenv APRILDEBUG false)" doc="if true launches the pose publisher node"/>
+	<arg name="rvizcfg" value="$(optenv RVIZCFG multi_robot)"/>
```

Comment:
Why don't you declare this directly in `spawn_multirobot`?

Reply:
Adressed